app/api-fetch/all-artists.ts

```
@@ -22,7 +22,7 @@
         throw new Error('API URL not set');
     }
     try {
-        if (localStorage.getItem("artists") !== null || Date.now() - parseInt(localStorage.getItem("artistsTime")!) < 3600000) {
+        if (localStorage.getItem("artists") !== null && Date.now() - parseInt(localStorage.getItem("artistsTime")!) < 3600000) {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)